bridge-history-api/internal/logic/l1_fetcher.go

```diff
@@ -118,6 +121,12 @@ func NewL1FetcherLogic(cfg *config.FetcherConfig, db *gorm.DB, client *ethclient
 		parser:          NewL1EventParser(cfg, client),
 	}
 
+	index, err := f.batchEventOrm.GetLastFinalizedBatchIndex(context.TODO())
```

Comment:
passing a context from parent call stack here?

Reply:
ok, will do that

Reply:
job is done